mail_edit/tests/test_ui.py

```diff
@@ -0,0 +1,15 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+import odoo.tests
+
+
+@odoo.tests.tagged("post_install", "-at_install")
+class TestMailEdit(odoo.tests.HttpCase):
+    def test_tour(self):
+        self.phantom_js(
+            "/",
```

Comment:
this should be '/web', otherwise you'll end up on the homepage, not then backend

Reply:
@hbrunn I have made the fix and did a rebase. Waiting for the runbot.